### PR TITLE
adds details from our recent Solr issues

### PR DESCRIPTION
**Pull request by acozine**
Adds information on diagnosing Solr issues as well as a description of the data center constraints. I hope this documentation will help us recover more quickly from future outages, and give everyone on the team some tools for diagnosing issues with our Solr infrastructure.

### CODE REVIEW

**Review by hectorcorrea — APPROVED** [review]

File: services/solr.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
+## Data center constraints for our Solr cloud
+Due to latency and throughput issues between our two data centers, all VMs on our Solr cloud must be in a single data center for indexing to run successfully. In an outage, one or more VMs may fail over to the other data center - read operations will continue in this scenario, but reindexing will fail. Once the data center has recovered from the outage, vMotion all VMs in our Solr cloud back to the same data center before reindexing.
```
Comment (maxkadel) [inline services/solr.md:2]: is there a missing word somewhere in `outage, vMotion all VMs in our Solr...`

Comment (maxkadel) [reply services/solr.md:2]: Or is vMotion a verb?

**Review by maxkadel — APPROVED** [review]
As long as vMotion is a verb, approved, otherwise fix typo and then I'm happy for it to be merged.

File: services/solr.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+To view the Solr admin UI, open an SSH tunnel to any VM in the Solr cloud (for example, `ssh -L 9000:localhost:8983 lib-solr-prod4`) then point a browser at the localhost port you defined for the tunnel: [localhost:9000](localhost:9000).
+
+The solr admin console shows (among other things):
+* details about the box you opened a tunnel to in the default `Dashboard`
+* performance on all VMs in the Solr cloud at `Cloud > Nodes`
+* the status of shards and replicas for all collections at `Cloud > Graph`
+* details on the configuration of each collection at `Collections > this-collection`
+* performance on all Zookeeper boxes at `Cloud > ZK status`
+
+To open the solr admin console, use the shortcut cap task in pul_solr: `bundle exec cap solr8-production solr:console`.
```
Comment (tpendragon) [inline services/solr.md:17]: We don't have a solr7 anymore, I think we can just do `bundle exec cap production solr:console`

**Review by tpendragon — APPROVED** [review]

File: services/solr.md (hunk 2 above)
Comment (acozine) [reply services/solr.md:17]: Hm, but we do have solr9 now . . . so we might still need this?

File: services/solr.md (hunk 1 above)
Comment (acozine) [reply services/solr.md:2]: Good catch - I have heard 'vMotion' used as a verb, but I think it would be better to avoid jargon here.

Comment (acozine) [reply services/solr.md]: addressed in 3c6b14ea159875924ef7f3becad3bcd1c0aabe95

File: services/solr.md (hunk 2 above)
Comment (acozine) [reply services/solr.md]: Addressed (I hope?) in 3c6b14ea159875924ef7f3becad3bcd1c0aabe95

**Review by maxkadel — APPROVED** [review]
Looks great, thanks @acozine!

**Review by tpendragon — APPROVED** [review]

**Pull request merged**